src/test/java/org/springframework/integration/kafka/listener/TestSingleBrokerExternal.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.springframework.integration.kafka.listener;
+
+import java.util.UUID;
+
+import org.junit.After;
+import org.junit.ClassRule;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import org.springframework.integration.kafka.rule.KafkaRule;
+import org.springframework.integration.kafka.rule.KafkaRunning;
+
+/**
+ * @author Marius Bogoevici
+ */
+@Ignore
```

Comment:
What do you mean by not finished? The test is set to be ignored right now just because it has the side effect of creating new topics (so it keeps writing data on disk), mainly because topic deletion doesn't actually work.

Unfortunately,there's no out of process equivalent for that method - that's why we have to resort to sleeping. That method is specifically designed to have access to the `KafkaServer` instance, which doesn't work outside the process. The amount of time for sleeping is really guesswork :(